code/datums/jobs/job/pmc.dm

```diff
@@ -68,6 +68,54 @@
 	H.equip_to_slot_or_del(new /obj/item/reagent_containers/hypospray/autoinjector/combat_advanced, SLOT_IN_HEAD)
 	H.equip_to_slot_or_del(new /obj/item/reagent_containers/food/snacks/enrg_bar, SLOT_IN_HEAD)
 
+//PMC medic
+/datum/job/pmc/medic
+	title = "PMC Medic"
+	paygrade = "PMC2"
+	skills_type = /datum/skills/combat_medic/pmc
+	outfit = /datum/outfit/job/pmc/medic
+
+//medic loadout
+/datum/outfit/job/pmc/gunner
+	name = "PMC Medic"
+	jobtype = /datum/job/pmc/medic
+
+	id = /obj/item/card/id/silver
+	belt = /obj/item/storage/belt/lifesaver/full
+	ears = /obj/item/radio/headset/distress/pmc
+	w_uniform = /obj/item/clothing/under/marine/veteran/pmc/webbing
```

Comment:
This is a black webbing, so idk how you intend on putting a gun in a storage that can't hold it...